src/keywords/CMakeLists.txt

```
@@ -40,6 +40,7 @@ add_library(
   vectorIntString.cpp
   vectorStringDouble.cpp
   vectorStringPair.cpp
+  rFactorRangesVector.cpp
```

We'll be able to use this new keyword in any context, so we should call it just
```suggestion
  rangeVector.cpp
```